boards/arm64/rpi_5/Kconfig.board

```diff
@@ -4,3 +4,7 @@
 config BOARD_RPI_5
 	bool "Broadcom BCM2712"
 	depends on SOC_BCM2712
+
+config BOARD_RPI_5_XEN_DOM0
```

Comment:
This whole commit is something improper. The basic idea of snippets - is extending or overwritting original board behavior: you use RPI5 and want to use Xen dom0 on it? Just apply `xen_dom0` overlay, no other changes or build targets should be needed. All Xen related actions should be done inside a snippet.

Reply:
I simply like this approach more as it allows to keep all platform data in one place, so added it as alternative to snippet. Np. will drop.

Reply:
removed and replaced with snippet